brion/spikeReport.h

```diff
@@ -53,7 +53,7 @@ class SpikeReport;
  *
  * This class is not thread-safe except where stated otherwise.
  */
-class SpikeReport : public boost::noncopyable
+class SpikeReport
```

Comment:
I would prefer to keep the boost::noncopyable: It makes clear from the beginning that the class is not copyable and I don't have to scroll down to the private section to understand that.
The noncopyable boost header is anyway included from <types.h>